packages/junokit/src/lib/compile/compile.ts

```diff
@@ -57,10 +60,10 @@ export async function compile (
 
   for (const dir of contractDirs) {
     compileContract(dir, docker);
+    const contractName = readContractName(path.join(dir, toml));
     if (!skipSchema) { // only generate schema if this flag is not passed
-      generateSchema(dir, docker);
+      await generateSchema(contractName, dir, docker);
```

Comment:
will this generate ts schema irrespective of which script is being used js or ts ?